.build/run-tests.sh

```diff
@@ -285,7 +285,7 @@ _main() {
   # check split_chunk is compatible with target (if not a regexp)
   if [[ "${_split_chunk}" =~ ^\d+/\d+$ ]] && [[ "1/1" != "${split_chunk}" ]] ; then
     case ${target} in
-      "stress-test" | "fqltool-test" | "sstableloader-test" | "microbench" | "cqlsh-test" | "simulator-dtest")
+      "stress-test" | "fqltool-test" | "sstableloader-test" | "microbench" | "cqlsh-test")
           error 1 "Target ${target} does not suport splits."
```

Comment:
```suggestion
          error 1 "Target ${target} does not support splits."
```
Not related to your patch but worth fixing the typo